pkg/server/middleware/middleware.go

```diff
@@ -87,7 +88,7 @@ func WithProviderToken(jwtClient auth.JWTClient, h http.Handler, log logr.Logger
 		tokenSlice := strings.Split(tokenStr, "token ")
 
 		if len(tokenSlice) < 2 {
-			log.Info("missing or invalid token.")
+			log.V(logger.LogLevelDebug).Info(fmt.Sprintf("missing or invalid token '%s'", tokenStr))
```

Comment:
I'm not sure if we should log out actual tokens. They might be slightly badly formatted or something but potentially still valid and a slight security risk.

Reply:
I think @foot makes a good point. What if instead of the token, we log the value of GitProviderTokenHeader instead - then I can look up the header in my request if I want to.